.github/workflows/code-quality.yaml

```
@@ -29,6 +29,17 @@ jobs:
             - run: yarn --immutable
             - run: yarn ci
 
+    coding-style-php:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v4
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '8.1'
```

```suggestion
                    php-version: '8.3'
```

I'm thinking tests are here to check our code works, but i'd rather use recent PHP versions on CI, for performance reasons and because it will be what most people will use locally too

wdyt ?

---

I believe you need to run php-cs-fixer on the lowest version of PHP you support. I've been burned in the past but maybe it's not a problem anymore.

---

That's better to run CS tools to the lowest supported PHP version. 
Some of them infers configuration based on running PHP version, and we don't want them to suggest more modern code that would break for lower PHP versions.

---

Ok then let's ignore :)